Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "==2.2.5"
+"psycopg2-binary" = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
Pipfile is not popular yet. 

I am not against it, but you probably should get used to requirements.txt file generated by

`pip freeze`

Reply:
Downgraded :)